kano_updater/commands/install.py

```diff
@@ -36,7 +36,7 @@ def install(progress=None):
     if status.state == UpdaterStatus.INSTALLING_UPDATES:
         msg = 'The install is already running'
         logger.warn(msg)
-        progress.abort(_(msg))
+        progress.abort(_('The install is already running'))
```

Comment:
@hzulla We definitely need to ensure that all the logging happens in English (to give us a hope of debugging it!) but this clearly isn't the ideal situation - having to define the strings twice. Do you know of any way of telling the logger to use the raw string but then telling `progress` to use the translated version?

I'm happy with what you have done here (and is indeed what I would do with my current knowledge of the translation engine) but in an ideal world we wouldn't need repeat the strings and only present the translated version when it hits the user.

Any ideas from anyone would be greatly appreciated @Ealdwulf @alex5imon @skarbat @pazdera @carolineclark @convolu